build_tools/python/e2e_test_framework/definitions/iree_definitions.py

```diff
@@ -79,7 +96,11 @@ class ModuleExecutionConfig(object):
 
 
 class MLIRDialectType(Enum):
-  """Imported MLIR dialect type."""
+  """Imported MLIR dialect type.
+
+  These strings are used as part of imported model id. They shouldn't be changed
```

Comment:
Hmmm that seems problematic. How about making the values integers instead?

Reply:
Create a map from `MLIRDialectType` to integer id.

The enum value in `MLIRDialectType` is used in other places for compilation flags (so yes, we shouldn't expect them not to change).